### DISCUSSION TITLE
markdown rendering issue

### DISCUSSION THREAD

**Discussion by ZZH-Finalize** [community/community#127344] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

It is hard to recognize the checkbox is checked or not.
![image](https://github.com/community/community/assets/33342565/7c2e59a4-85fb-4c73-9879-b0002f48e554)

What changes are you suggesting?
Modify the checkbox style like this.
![image](https://github.com/community/community/assets/33342565/1456edb7-5600-41d0-b5d9-bb0511962e7c)

**Comment by 0xshubhs** [comment]
Hi ZZH-Finalize,

I understand the difficulty in recognizing whether the checkbox is checked or not in the current markdown rendering.

**Suggested Change:**

To improve visibility, modify the checkbox style like this:

In here you can recognize it
```markdown
- [ ] Unchecked
- [x] Checked
```
It will look like this in md
- [ ] Unchecked
- [x] Checked

but if you want the color to be changed 
```markdown
- [ ] Unchecked
- <span style="color: green;">&#9745;</span> Checked
```
it will look like this
- [ ] Unchecked
- <span style="color: green;">&#9745;</span> Checked
> With this you can change the color also if you want

I hope this helps
#127344

**Reply by ZZH-Finalize** [reply]
This is useful, but what I want is to change the default style of the checked checkbox in the document scope or global scope to prevent writing HTML tags every time.

**Reply by 0xshubhs** [reply]
Unfortunately, Markdown itself doesn't provide direct mechanisms for customizing the styling of checkboxes globally across a document. Markdown is designed to be a lightweight markup language and doesn't support advanced styling options like CSS.

However, if you're using Markdown in a context that supports HTML and CSS, such as some Markdown editors or platforms like GitHub, you can use a bit of HTML and inline CSS to style checkboxes. Here's how you can do it:

```html
<input type="checkbox" style="color: green;" disabled /> Checked
```

In this example:
- `disabled` is used to prevent users from toggling the checkbox.
- `color: green;` is the inline CSS that changes the color of the checkbox.

This approach allows you to change the default style of the checked checkbox without writing HTML tags every time.

If you're using Markdown in a specific tool or platform, please let me know so I can provide more specific advice.

**Reply by ZZH-Finalize** [reply]
I just want to do this on Github markdown documents like readme.md, but this trick seems to not work.

```markdown
<input type="checkbox" style="color: green;" disabled /> Checked

- [ ] unchecked
- [x] checked
```

<input type="checkbox" style="color: green;" disabled /> Checked

- [ ] unchecked
- [x] checked

Screenshot from my document:
![image](https://github.com/community/community/assets/33342565/bdc6ff67-36e5-42d9-8996-b4763b3e1d54)

**Reply by 0xshubhs** [reply, marked as answer]
```markdown
:heavy_check_mark: checked |
:white_check_mark: checked | 
:heavy_check_mark: checked |
```
This can change to this 
:heavy_check_mark: checked |
:white_check_mark: checked | 
:heavy_check_mark: checked |

I hope this helps 
Mark it as answer if you found this as an answer

**Reply by ZZH-Finalize** [reply]
Well, this could be an alternative solution, thanks for your help.

**Reply by 0xshubhs** [reply]
Yeah right!! Found this on stackoverflow!!
